subsys/bluetooth/host/gatt.c

```diff
@@ -2539,23 +2594,169 @@ int bt_gatt_notify_cb(struct bt_conn *conn,
 }
 
 #if defined(CONFIG_BT_GATT_NOTIFY_MULTIPLE)
+static int gatt_notify_multiple_valid_args(struct bt_conn *conn, uint16_t num_params,
+					   struct bt_gatt_notify_params *params)
```

Comment:
Typically the array comes before the size, and it may be easier to determine that it is an array by using `[]` instead of `*`

```suggestion
static int gatt_notify_multiple_valid_args(struct bt_conn *conn, const struct bt_gatt_notify_params params[], uint16_t num_params)
```

Reply:
yep, I was just trying to match the arg order of `bt_gatt_notify_multiple`.